pages/common/pup.md

```diff
@@ -0,0 +1,27 @@
+# pup
+
+> HTML parsing tool.
+
+- Transform raw HTML file into a cleaned, indented, and colored format:
+
+`cat {{index.html}} | pup --color`
+
+- Filter HTML by element tag name:
+
+`cat {{index.html}} | pup '{{tag}}'`
+
+- Filter HTML by id:
+
+`cat {{index.html}} | pup '{{tag#id}}'`
+
+- Filter HTML by attribute value:
+
+`cat {{index.html}} | pup '{{tag[attribute="value"}}'`
```

Comment:
Change the filter value to `input[type="text"]` to give another concrete example.